modules/git/bootstrap.Makefile

```diff
@@ -4,28 +4,28 @@ ifeq ($(wildcard .git),)
   endif
 else
 
-GIT ?= $(shell which git)
+GIT := $(shell which git)
 
-export GIT_COMMIT ?= $(shell $(GIT) rev-parse --verify HEAD 2>/dev/null)
```

Comment:
How come only this module is affected by the make 4.4 issue and not other modules that also use the `?=` format?

Reply:
it could be that it's a behavior in how `bootstrap` level Makefiles get imported.